eduNEXT/platform-plugin-forum-email-notifier
# fix: add `locale` folder in `MANIFEST.in`

### Description
This PR adds the `locale` folder into the `MANIFEST.in` file, this will add the files that are inside this directory at the time of package installation.

### How to Test
1. Uninstall this plugin in your environment with `tutor dev exec lms pip uninstall platform-plugin-forum-email-notifier -y`
2. Install the plugin without the `-e` flag: `tutor dev exec lms pip install /openedx/requirements/platform-plugin-forum-email-notifier`
3. Restart the environment: `tutor dev restart`
4. Go to the instructor dashboard
5. Select the forum notifications tab
6. You should see all text translated into your preferred language
